bundles/core/org.eclipse.smarthome.core.thing/src/main/java/org/eclipse/smarthome/core/thing/Thing.java

```diff
@@ -23,9 +24,25 @@
  * through a {@link Bridge}.
  *
  * @author Dennis Nobel - Initial contribution and API
+ * @author Thomas Höfer - Added thing and thing type properties
  */
 public interface Thing {
 
+    /** the key for the vendor property */
```

Comment:
Why did you decide to move the constants in the Thing interface? I think we could have kept the CommonPropertyKey enumeration as well. Having them in the Thing itself makes them look as if they are mandatory or specifically important. If they were, one would rather expect individual fields for them (what I am not suggesting, don't get me wrong!)